pkg/autoscaling/prometheus.go

```diff
@@ -164,3 +176,35 @@ func buildCPUUsagePromQL(options *QueryOptions) (string, error) {
 	query := fmt.Sprintf(pattern, options.duration.String())
 	return query, nil
 }
+
+// this function assumes that addr is already a valid resolvable address
+// returns in format "podname_namespace"
+func getInstanceNameFromAddress(addr string) string {
```

Comment:
As the address here is special (made by tidb-cloud's own monitor rule), I suggest check the format here, and if it not match the format we want, just throwing an error instead of throwing the origin address.

Reply:
Is the format `pod-name.peer-svc.namespace.svc`?

Reply:
updated